### DISCUSSION TITLE
New to standardNotes

### DISCUSSION THREAD

**Discussion by CyberSmurfs** [standardnotes/forum#3346] [General]
Hi

I am new user to SN and I have the free version of i but thinking of upgrade to productivity or Professional but which one of them are the best choise and which one do I get the most out of?

**Comment by effieeee** [comment]
Hi there, welcome to SN! Personally I would suggest trying the Professional plan if you can, since this is the optimal experience to get the most out of the app 👌 Note that there's also a [100% refund guarantee](https://standardnotes.com/help/51/what-is-the-refund-policy-on-standard-notes-extended) if you ever find that the paid features don't quite suit your needs!

**Reply by CyberSmurfs** [reply]
Hi
Sorry for late reply. Thanks. I am thinking of upgrade to it but when I look on the Net and convert 60 USD to SEK I get 627 swedish crona wich is quite expensive I think but I´ll think about it and make a decision.
